tests/unit/helpers/abi_test_x64.h

```diff
@@ -0,0 +1,5669 @@
+              Size:            8
+    PreservedRegisters:
+      - rbx_x86_64
+      - rbp_x86_64
+      - rsi_x86_64
+      - rdi_x86_64
+      - r12_x86_64
+      - r13_x86_64
+      - r14_x86_64
+      - r15_x86_64
+      - xmm6_x86_64
+      - xmm7_x86_64
+    FinalStackOffset: 16
+  - !RawFunctionType
+    Kind:            RawFunctionType
+    ID:              2000000044
+    Arguments:
+      - Location:        rcx_x86_64
+        Type:
+          UnqualifiedType: "/Types/Primitive-776"
+      - Location:        rdx_x86_64
+        Type:
+          UnqualifiedType: "/Types/Primitive-776"
+      - Location:        r8_x86_64
+        Type:
+          UnqualifiedType: "/Types/Primitive-776"
+    ReturnValues:
+      - Location:        rax_x86_64
+        Type:
+          UnqualifiedType: "/Types/Struct-7000000007"
+          Qualifiers:
+            - Kind:            Pointer
+              Size:            8
+    PreservedRegisters:
+      - rbx_x86_64
+      - rbp_x86_64
+      - rsi_x86_64
+      - rdi_x86_64
+      - r12_x86_64
+      - r13_x86_64
+      - r14_x86_64
+      - r15_x86_64
+      - xmm6_x86_64
+      - xmm7_x86_64
+    FinalStackOffset: 0
+  - !RawFunctionType
+    Kind:            RawFunctionType
+    ID:              2000000045
+    Arguments:
+      - Location:        rcx_x86_64
+        Type:
+          UnqualifiedType: "/Types/Primitive-776"
+      - Location:        rdx_x86_64
+        Type:
+          UnqualifiedType: "/Types/Primitive-776"
+      - Location:        r8_x86_64
+        Type:
+          UnqualifiedType: "/Types/Primitive-776"
+      - Location:        r9_x86_64
+        Type:
+          UnqualifiedType: "/Types/Primitive-776"
+    ReturnValues:
+      - Location:        rax_x86_64
+        Type:
+          UnqualifiedType: "/Types/Struct-7000000010"
+          Qualifiers:
+            - Kind:            Pointer
+              Size:            8
+    PreservedRegisters:
+      - rbx_x86_64
+      - rbp_x86_64
+      - rsi_x86_64
+      - rdi_x86_64
+      - r12_x86_64
+      - r13_x86_64
+      - r14_x86_64
+      - r15_x86_64
+      - xmm6_x86_64
+      - xmm7_x86_64
+    FinalStackOffset: 0
+Segments:        []
+Functions:       []
+ImportedDynamicFunctions: []
+...
+)";
+
+inline const SortedVector<size_t> Microsoft_x64_vectorcall_IDs{
+  1000000001, 1000000004, 1000000014, 1000000019, 1000000020, 1000000021,
+  1000000022,
+
+  2000000001, 2000000002, 2000000003, 2000000004, 2000000005, 2000000006,
+  2000000007, 2000000008, 2000000009, 2000000010, 2000000011, 2000000012,
+  2000000013, 2000000014, 2000000015, 2000000016, 2000000017, 2000000018,
+  2000000019, 2000000020, 2000000021, 2000000022, 2000000023, 2000000024,
+  2000000025, 2000000026, 2000000027, 2000000028, 2000000029, 2000000030,
+  2000000031, 2000000032, 2000000033, 2000000034, 2000000035, 2000000036,
+  2000000037, 2000000038, 2000000039, 2000000040, 2000000041, 2000000042,
+  2000000043, 2000000044, 2000000045
+};
```

Comment:
Mmmmh, what's this?

Reply:
As to why there are so many `2...` functions - that's because it's the CABI -> Raw conversion which should succeed pretty much always, should it not?
=